node/src/proxy_server/mod.rs

```diff
@@ -2537,20 +2479,23 @@ mod tests {
             originator_public_key: PublicKey::new(b"originator_public_key"),
         };
         let logger = Logger::new("test");
+        let local_tth_args = TTHArgsLocal {
+            common: TTHArgsCommon {
+                cryptde,
+                payload,
+                source_addr,
+                timestamp: SystemTime::now(),
+                is_decentralized: false,
```

Comment:
Again, I'm a bit nervous about this. If tracing through the code indicates that nothing important is cut out by this, okay, but I'd want to be sure.